getgather/mcp/dpage.py

```diff
@@ -241,10 +260,26 @@ async def post_dpage(id: str, request: Request) -> HTMLResponse:
                             value = fields.get(name_str)
                             if not value or len(value) == 0:
                                 logger.warning(f"No form data found for radio button group {name}")
+                                await report_distill_error(
```

Comment:
This needs to be excluded from error reporting for now.
The lack of `name` is a mistake when crafting the distillation pattern (usually pretty quickly discovered during testing), nothing to do with the actual page and certainly the screenshot won't help.